Adds persistence and sending of feedback

eirism opened the pull request:
Lecture sessions and counts of each feedback type are saved in the DB.
Adds socketio handlers to process incoming events from students and lectures.

eirism committed:
Adds persistence and sending of feedback

Lecture sessions and counts of each feedback type are saved in the DB.
Adds socketio handlers to process incoming events from students and lectures.

sardred commented on config.py:
```diff
@@ -6,3 +6,10 @@
 SQLALCHEMY_MIGRATE_REPO = os.path.join(basedir, 'db_repository')
+
+BUTTON_ACTIONS = [
+    ['action_slow', 'To slow'],
```
Too*

plusk commented on config.py:
```diff
@@ -6,3 +6,10 @@
 SQLALCHEMY_DATABASE_URI = os.environ['DATABASE_URL']
 SQLALCHEMY_MIGRATE_REPO = os.path.join(basedir, 'db_repository')
+
+BUTTON_ACTIONS = [
```
This should be more consistent. Either say "Too slow, too fast, too easy, too hard", or be more verbose and say "this is too _something_".

plusk replied:
This is a discussion we should take as a group too, considering I'd personally prefer just saying "fast, slow, easy, hard", but considering this is a "bot", we might want to be more communicative with the user.

plusk reviewed (dismissed):
Other than this, it looks alright.

eirism committed:
JSON scheme changed, updated button titles

The new scheme remeoves redundancy.
It was agreed upon that short titles is the way to go, for now
The chat.html is updated with testing stuff, will be refactored
and moved to appropiate pages/files
